src/WebJobs.Script/Diagnostics/Extensions/ScriptHostLoggerExtension.cs

```diff
@@ -95,6 +95,12 @@ internal static class ScriptHostLoggerExtension
             new EventId(413, nameof(ScriptHostStarted)),
             "Host started ({ms}ms)");
 
+        private static readonly Action<ILogger, Exception> _addingDescriptorProviderForHttpInvoker =
+           LoggerMessage.Define(
+           LogLevel.Debug,
+           new EventId(414, nameof(AddingDescriptorProviderForHttpInvoker)),
+           "Creating function descriptors.");
```

Comment:
Why do we need this copy - we already have a log value _creatingDescriptors. Just reuse that?

Reply:
fixed logging here. _creatingDescriptors is used for adding all descriptors. AddingDescriptorProviderForHttpWorker is for logging specifically HttpWorker descriptor